src/Mailer/Email.php

```diff
@@ -1484,6 +1485,17 @@ protected function _encode($text)
     }
 
     /**
+     * Decode the specified string
+     *
+     * @param string $text String to decode
+     * @return string Decoded string
+     */
+    protected function _decode($text)
+    {
+        return mb_decode_mimeheader($text);
```

Comment:
I don't think we necessarily need them. This method has one caller, which is `getOriginalSubject()`. If that method has tests, then this method is 'covered' and could even be inlined.